docs/developer/standards.md

```diff
@@ -0,0 +1,61 @@
+# Standards
+
+The following standards are used by FAST-DNA contributors for consistency across this repository.
+
+## Core principles
+
+* Build for maintainability
+* Build for scalability
+* Build for performance
+
+## Naming Conventions
+
+* Avoid all abbreviations, except where common industry acronyms
+* Use names that are clear and intuitive
+* Use semantic names instead of presentational
+
+## Spacing
```

Comment:
technically, anything regarding code style can be removed. If prettier cares about it, prettier will address it.